deployment/cdk/config.py

```diff
@@ -99,7 +100,7 @@ class Config:
         """model config"""
 
         env_file = "deployment/.env"
-        env_prefix = "EOAPI_RASTER_"
+        env_prefix = "CDK_EOAPI_RASTER_"
```

Comment:
This is to avoid confusion between `EOAPI_RASTER_` env used in `eoAPI.raster` module and the ones used for deployment.